curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables.md

```diff
@@ -32,10 +34,12 @@ function propPrefix(str) {
   var s = "prop";
   return s + str;
 }
-var someProp = propPrefix("Name"); // someProp now holds the value 'propName'
-console.log(someObj[someProp]); // "John"
+var someProp = propPrefix("Name");
+console.log(someObj[someProp]);
 ```
 
+`someProp` has a value of the string `propName`, and the string `John` would be displayed in the console.
```

Comment:
Again, the voice is changing now to `has`...

Reply:
`will have`? Or `holds`?

Reply:
I do not mind, just mentioning that it is changing. It can stay as is - I am nitpicky